app/src/androidTest/java/org/mozilla/tv/firefox/ui/BasicNavigationTest.kt

```diff
@@ -141,4 +142,5 @@ class BasicNavigationTest {
             assertNotEquals(preReloadTitle, getPageTitle())
         }
     }
+    /* ktlint-enable no-blank-line-before-rbrace */
```

Comment:
Does this work if we put the suppression at the top of the file, but nothing at the bottom?  It would be really easy to miss this closing comment.

Reply:
Yep. I thought putting it directly above the test though was helpful for people copying this test format to know how to suppress and that it's more clear *why* this was necessary. I'll remove the bottom comments but leave the top one above the tests.